persistence-test/src/main/java/io/stargate/it/storage/ExternalStorage.java

```diff
@@ -61,6 +64,17 @@ public ExternalStorage() {
     super(ClusterSpec.class, STORE_KEY, Namespace.GLOBAL);
   }
 
+  private Cluster createCluster(ClusterSpec spec, ExtensionContext context) {
```

Comment:
could you show an example of how this new customization will be used from the caller's perspective?

Reply:
Doesn't `CcmCluster` provide an example?